src/os/ui/config/betatoggleui.js

```diff
@@ -0,0 +1,101 @@
+goog.declareModuleId('os.file.BetaToggleUI');
```

Comment:
Same comment as above here, also should be called `betatogglesettingsui.js` (If I'm being picky :) )